users/views.py

```diff
@@ -110,40 +115,44 @@ def login_process(request):
 
 def signup_view(request):
 	form = SignupForm()
-	context = {'form': form, 'result': request.GET.get('result')}
+	context = generate_signup_form(form)
+	context['result'] = request.GET.get('result')
 	return render(request, 'users/signup.html', context)
 
+def generate_signup_form(form):
```

Comment:
maybe a name like add_captcha_to_form would be better

Reply:
Done